### PR TITLE
lnrpc+rpcserver: expose short chan id on feereport

### PR DESCRIPTION
**Pull request by cfromknecht**
This commit adds each channel's short chan id to the `feereport` rpc.
Without this, it can be tedious to lookup more info about a particular
channel since most rpcs only accept short chan ids and not channel
points. For instance, now one can take a channel id from `feereport` and
look it up directly via `getchaninfo` to examine the policy in more detail.

### CODE REVIEW

File: lnrpc/rpc.proto [hunk 1]
```diff
@@ -2895,6 +2895,9 @@ message Feature {
 
 message FeeReportRequest {}
 message ChannelFeeReport {
+    /// The short channel id that this fee report belongs to.
```
Comment (carlaKC) [inline lnrpc/rpc.proto]: Is this at the top of the message for the sake of lncli? If so, rather leave the proto in order and add an ad-hoc struct which prints the order we want in lncli?

**Comment by joostjager** [comment]
Two reviewers already, unassigning myself

File: lnrpc/rpc.proto (hunk 1 above)
Comment (cfromknecht) [reply lnrpc/rpc.proto]: It’s not only for lncli, but also rest. In 0.9 we went through and removed a bunch of structs that were purely for the purpose of formatting so I’m hesitant to start down that road again

Comment (cfromknecht) [reply lnrpc/rpc.proto]: fwiw the rationale for doing so is that `jsonpb` now automatically converts byte slices to hex, but it only operates on `proto.Message` objects. if we had custom formatting they'd either have to implement this interface or add back this custom serialization, but all of this is avoided by just reordering the protos.

Comment (carlaKC) [reply lnrpc/rpc.proto]: Ah I see, good to know!

**Pull request merged**